docs/CHANGELOG.md

```diff
@@ -10,10 +10,14 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 ## [Unreleased]
 
 ### Added
+
 * Fixes and renames regression from ([#1442]) added back `position_from_world` as `screen_to_world`. Also added
 `world_to_screen`. Also adds `Transform::copy_local_to_global()' for `debug_assertion` builds ([#1733])
+* Add `add_square`, `add_rotated_square`, `add_box`, `add_rotated_box`, `add_circle`, `add_rotated_circle`, `add_sphere` functions
+to `DebugLinesComponent` and the corresponding draw functions to `DebugLines` to draw simple shapes with debug lines. ([#1651])
 
 ### Changed
+
```

Comment:
Add a link to this pull request, like other pull requests are doing near this line https://github.com/amethyst/amethyst/pull/1766/files#diff-81baa3f7e3d2d55c8a408032bf865b0fR25

Thanks! :)

Reply:
Hmm, I somehow thought it did that automatically...